tests/model_inheritance/tests.py

```diff
@@ -222,6 +223,36 @@ def test_queryset_class_getitem(self):
         self.assertIs(models.QuerySet[Post, Post], models.QuerySet)
         self.assertIs(models.QuerySet[Post, int, str], models.QuerySet)
 
+    def test_shadow_parent_attribute_with_field(self):
+        class ScalerParent(models.Model):
+            foo = 1
+
+        class ScalerOverride(ScalerParent):
+            foo = models.IntegerField()
+
+        self.assertEqual(type(ScalerOverride.foo), DeferredAttribute)
```

Comment:
```suggestion
        class ScalarParent(models.Model):
            foo = 1

        class ScalarOverride(ScalarParent):
            foo = models.IntegerField()

        self.assertEqual(type(ScalarOverride.foo), DeferredAttribute)
```

Reply:
That's Bristol that is.

Reply:
I don't know what you're saying but it sounds rude in cockney rhyming slang... https://en.wiktionary.org/wiki/bristol#Noun